src/applications/personalization/profile/util/index.js

```diff
@@ -70,6 +70,26 @@ export const hasInvalidWorkPhoneNumberError = errors =>
 export const hasPaymentRestrictionIndicatorsError = errors =>
   hasErrorMessage(errors, PAYMENT_RESTRICTIONS_PRESENT_KEY);
 
+export const directDepositBankInfo = apiData => {
```

Comment:
These three functions all contain functionality that used to live only in the `profile/selectors.js` file. But the logic that digs into the response from `GET payment_information` needs to also be used in the `paymentInformation` actions file. These helpers can now be used in the `actions`, to pull data directly out of the data we get from the API, and in the `selectors` to work with the API data after it's been stored in Redux.